contracts/LpSugar.vy

```diff
@@ -142,6 +142,23 @@ struct AlmAddresses:
   farm: address
   lp_wrapper: address
 
+struct AlmImmutableParams:
+   core: IALMCore
+   strategyModule: address
+   veloModule: address
+   depositWithdrawModule: address
+   helper: address
+
+struct AlmManagedPositionInfo:
```

Comment:
Would be great to have links to where it is referenced, eg:
https://github.com/mellow-finance/mellow-alm-toolkit/blob/main/src/interfaces/ICore.sol#L12-L18